tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateKeyboardCharts.java

```diff
@@ -0,0 +1,66 @@
+package org.unicode.cldr.tool;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import org.unicode.cldr.util.CLDRPaths;
+
+public class GenerateKeyboardCharts {
+
+    public static void main(String args[]) throws IOException {
+        final File mainDir = new File(CLDRPaths.CHART_DIRECTORY);
+        if (mainDir.mkdirs()) {
+            System.err.println("Created: " + mainDir);
+        }
+        if (!mainDir.isDirectory()) {
+            throw new IOException("Main dir doesn't exist: " + mainDir);
+        }
+        final File kbdDir = new File(CLDRPaths.BASE_DIRECTORY, "docs/charts/keyboard");
+        if (!kbdDir.exists()) {
+            throw new IOException("Keyboards root dir doesn't exist: " + kbdDir);
+        }
+        final File kbdStatic = new File(kbdDir, "static");
+        final File kbdStaticData = new File(kbdDir, "static/data");
+        if (!kbdStaticData.exists()) {
+            System.err.println(
+                    "ERROR: " + kbdStaticData + " does not exist. Keyboard charts weren't run.");
+            System.err.println("See " + new File(kbdDir, "README.md") + " for help.");
```

Comment:
@macchiati this only produces a warning, but the readme says how to build the charts